Account/FirefoxAccountConfiguration.swift

```diff
@@ -27,7 +30,7 @@ public enum FirefoxAccountConfigurationLabel: String {
  * and context=fx_ios_v1 opts us in to the Desktop Sync postMessage interface.
  */
 public protocol FirefoxAccountConfiguration {
-    init()
+    init(prefs: Prefs?)
```

Comment:
Remove both of these lines, then you can just use the constructor you need for each type (you don't have to use the same one for all types).